PSDesiredStateConfiguration.md

```diff
@@ -0,0 +1,106 @@
+## Getting Started
+
+**VMware.PSDesiredStateConfiguration** provides a new way to compile and execute **DSC Configurations** on **Windows**, **Linux** and **MacOS** that does not require the use of **DSC LCM** or the typical MOF format and without altering the existing **Configuration** syntax.  Instead the cmdlets of this module create and work with regular powershell objects. This module is also designed to work with the existing **DSC Resources** in **VMware.vSphereDSC**.  
+
+## Requirements
+
+The following table describes the required dependencies for running VMware.vSphereDSC Resources.
+
+ **Required dependency**   | **Minimum version**
+-------------------------- | -------------------
+`PowerShell`               | 5.1 (Core is supported)
+
+1. Copy the VMware.PSDesiredStateConfiguration Module to one of the system PowerShell module directories. For more information on installing PowerShell Modules, please visit [Installing a PowerShell Module](https://docs.microsoft.com/en-us/powershell/module/powershellget/install-module?view=powershell-7).
+2. In PowerShell import the VMware.PSDesiredStateConfiguration Module:
+   ```
+    Import-Module -Name 'VMware.PSDesiredStateConfiguration'
+   ```
+
+   To check if the module was successfully installed:
+   ```
+    Get-Module -Name 'VMware.PSDesiredStateConfiguration'
+   ```
+This module also has additional requirements depending on which version of powershell you wish to use it with.
+
+#### Powershell core requirements
+---
+By using **Powershell Core** this module depends on the Invoke-DscResource cmdlet and it must be enabled with the following command.
+```
+Enable-ExperimentalFeature PSDesiredStateConfiguration.InvokeDscResource
+```
+ After executing the command you must restart your powershell session. The command needs to be run only once and the cmdlet will be usable.
+ To check if the Invoke-DscResource cmdlet has been enabled correctly run the following command
+```
+Get-Module 'PSDesiredStateConfiguration' -ListAvailable | Select-Object -ExpandProperty ExportedCommands
+```
+##### Important!
+When using **Powershell core** please keep in mind the list of [Known Limitations](https://github.com/vmware/dscr-for-vmware/blob/master/LIMITATIONS.md)
+
+---
+#### Powershell 5.1 requirements
+For **Powershell 5.1** you need to enable Windows Remote Management
+For information on how to enable it read here: [WinRM guide](https://docs.microsoft.com/en-us/windows/win32/winrm/installation-and-configuration-for-windows-remote-management)
+
+# Guide
+#### New-VmwDscConfiguration
```

Comment:
Before stepping to cmdlets one by one. Add an overview section. VMware.PSDesiredStateConfiguration works with ps1 configuration files. The flow of set, get, and test configuration is as follows
1. Compile ps1 DSC configuration. Transforms given configuration file to a configuration object (give details about the PS Object instance). Mention MOF doesn;t play a role in this stage
2. Pass configuration object Get Set Test cmdlets